commit 44a43411ec05132e06a091ceeaa928a34e6d203b (refs/changes/60/4860/1)
Author: Mike Gerdts <mike.gerdts@joyent.com>
Date:   2018-09-21T17:08:57+00:00 (1 year, 1 month ago)
    
    OS-7258 sysinfo needs to indicate if a PI can run kvm and bhyve concurrently

diff --git a/src/sysinfo b/src/sysinfo
index 106da2ba..79239247 100644
--- a/src/sysinfo
+++ b/src/sysinfo
@@ -219,6 +219,7 @@ function get_bhyve_capability()
             if [[ -n ${vcpus} ]]; then
                 Bhyve_Max_Vcpus=${vcpus}
             fi
+	    HVM_API="true"
         fi
     fi
 }
@@ -826,6 +827,7 @@ END
   "VM Capable": ${VM_Capable},
   "Bhyve Capable": ${Bhyve_Capable},
   "Bhyve Max Vcpus": ${Bhyve_Max_Vcpus},
+  "HVM API": ${HVM_API},
   "CPU Type": "${CPU_Version}",
   "CPU Virtualization": "${CPU_Virtualization}",
   "CPU Physical Cores": ${CPU_Count},
